doc/source/whatsnew/v0.20.0.txt

```diff
@@ -75,6 +75,21 @@ Strings passed to ``DataFrame.groupby()`` as the ``by`` parameter may now refere
 
    df.groupby(['second', 'A']).sum()
 
+.. _whatsnew_0200.enhancements.groupby_ngroup:
+
+Groupby Group Numbers
+^^^^^^^^^^^^^^^^^^^^^
+
+A new groupby method ``ngroup``, parallel to the existing ``cumcount``, has been added to return the group order (:issue:`11642`).
```

Comment:
or even better: link to their docstring page, using ``` :meth:`~pandas.core.groupby.GroupBy.ngroup`  ``` (will only display the ngroup)